### DISCUSSION TITLE
Difference between medusa's and medusa-extender's migration cli

### DISCUSSION THREAD

**Discussion by seanwyndellque** [adrien2p/medusa-extender#98] [Q&A] [answered]
Hi, is there a difference between `medusa migrations run` and `medex m --run`? I noticed that on initial set up of medusa with medusa-extender, I still need to run `medusa migrations run`. When should I use one over the other? Are they interchangeable?

**Comment by adrien2p** [comment, marked as answer]
Hi there, the extender cli is made for the extender architecture and components generations that makes your life easier (since the architecture is more flexible and a bit different from the one expected from medusa, that can still be used btw, it helps you with that part). The migration command requires you to setup the different paths that you want typeorm to pull your migration files from. In other word, it is not meant to replace the one from medusa that stay your main cli, but it helps you using the extender with less frictions 😀 hope that helps you.
